test/stdlib/Kernel_test.rb

```diff
@@ -888,6 +888,17 @@ def test_respond_to?
     obj.respond_to?('to_s', true)
   end
 
+  def test_respond_to_missing?
+    obj = Object.new
+    
+    # The default implementation always returns `false` regardless of the args,
+    # let alone their types; though overrides only have to support Symbol + bool
+    assert_send_type(
+      "(Symbol, bool) -> bool",
+      obj, :respond_to_missing?, :to_s, true
```

Comment:
I see other `(bool)` tests simply use `bool` for the type and `true` for the arg, instead of separate tests for true and false. I suppose I’ll match.